account/user.go

```diff
@@ -31,13 +31,18 @@ type User struct {
 	Bio                string             // The bio of the User
 	URL                string             // The URL of the User
 	Company            string             // The (optional) Company of the User
-	FeatureLevel       *string            // the (optional) level of features that the user opted in (to access unreleased features). Defaults to `released` so no non-released feature is enabled for the user.
+	FeatureLevel       string             // the (optional) level of features that the user opted in (to access unreleased features). Defaults to `released` so no non-released feature is enabled for the user.
```

Comment:
Fix the comment pleas. It's not optional anymore :)

Reply:
oh yes, you're right! thanks for spotting that.